sorting/bubbleSort.js

```diff
@@ -0,0 +1,21 @@
+const bubbleSort = (arr) => {
+
+    if(arr !== null) {
+        let len = arr.length;
+
+        let temp = 0;
+        for (let i=0; i<len; i++) {
+            for (let j=0; j<len-i-1; j++) {
+                if (arr[j] > arr[j+1]) {
+                    temp = arr[j+1];
```

Comment:
this is hard to read, create a separate function for swapping.
pro-tip: use destructuring to swap elements
```javascript
[arr[i],arr[j]] = [arr[j],arr[i]]
```